code-challenges.test.js

```diff
@@ -16,35 +16,140 @@
 // a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
+describe("arrayShuffle", () => {
+    it("removes the first item from the array and shuffles the remaining content", () => {
+      expect(arrayShuffle(colors1)).toEqual(["yellow", "blue", "pink", "green"])
```

Comment:
Since our goal is to rearrange the order of the array, .toEqual by itself will check only strict equality.  We need to add some things into our expect statement to check for a variation of the array.